getting_started/openshift-overview.adoc

```diff
@@ -0,0 +1,87 @@
+[id="openshift-overview"]
+= Overview
+include::modules/common-attributes.adoc[]
+:context: openshift-overview
+
+toc::[]
+
+{product-title} is a container orchestration platform based on Kubernetes. It benefits both operations and development. {product-title} provides developers and IT organizations with cloud application platforms that can be used for deploying applications on secure and scalable resources. It requires minimal configuration and management overhead.  {product-title} is built on Kubernetes. {product-title} brings Kubernetes platform to customer data centers and cloud. It meets security, privacy, compliance, and governance requirements.
+
+include::modules/getting-started-openshift-common-terms.adoc[leveloffset=+1]
+
```

Comment:
For Reviewers, 
Would it make sense to add a Prerequisite section that details what a user should have done prior to getting started? For example, downloaded and installed OpenShift? I would have loved to have something similar to this, but I'm not sure if this is possible? https://docs.openshift.com/container-platform/3.11/getting_started/install_openshift.html and https://docs.openshift.com/container-platform/3.11/getting_started/configure_openshift.html

Reply:
@sagidlow , +1 to creating a prerequisite section, but it may need to be pretty high level to accommodate all of the various OpenShift deployment scenarios.

Reply:
In both walk through we have a "Before you begin", for 4.x probably is sufficient. In Web Console Walkthrough could we have a table with the supported browsers and versions?

Reply:
@sagidlow, it makes total sense, but I agree with @sjstout, because there was major changes from 3.11 to 4.x regarding installation, so a high level overview about prereqs and install options would be nice.